### PR TITLE
Enable WSL->Windows fallback for open.

### PR DESCRIPTION
**Pull request by davidchisnall**

### CODE REVIEW

**Comment by davidchisnall** [comment]
Bonus points please!

File: meeting-backgrounds.py [hunk 1]
```diff
@@ -169,7 +169,13 @@ def open_folder(path):
     elif platform.system() == "Darwin":
         subprocess.Popen(["open", path])
     else:
-        subprocess.Popen(["xdg-open", path])
+        try:
+            subprocess.Popen(["xdg-open", path])
+        except FileNotFoundError:
+            if (os.environ["WSL_DISTRO_NAME"] is None):
```
Comment (letmaik) [inline meeting-backgrounds.py]:
```suggestion
            if os.environ.get("WSL_DISTRO_NAME") is None:
```

**Review by letmaik — APPROVED** [review]
Looks good, just one minor issue to fix.

**Comment by letmaik** [comment]
I hereby transfer ownership of these bonus points to you: ⭐🌟✨💫.

**Pull request merged**